src/views/webview-app/components/resources-panel/resources-panel.less

```diff
@@ -0,0 +1,124 @@
+
+.resources-panel {
+  position: fixed;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  z-index: 10;
+}
+
+.resources-panel-background {
+  position: fixed;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  z-index: -1;
+  background-color: rgba(0, 0, 0, 0.5);
+  animation-duration: 250ms;
+  animation-direction: forwards;
+  animation-name: backgroundDarkenAnimation;
+}
+
+@keyframes backgroundDarkenAnimation {
+  from {
+    width: rgba(0, 0, 0, 0.1);
+  }
+
+  to {
+    width: rgba(0, 0, 0, 0.5);
+  }
+}
+
+.resources-panel-content {
+  position: absolute;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  width: 420px;
+  padding: 48px 24px;
+  background: var(--vscode-editor-background);
+  width: 420px;
+  animation-duration: 250ms;
+  animation-direction: forwards;
+  animation-name: panelOpenAnimation;
+  animation-timing-function: cubic-bezier(0, 1.3, 0.7, 1);
+  box-shadow: -4px 0px 5px rgba(0, 0, 0, 0.25);
+}
+
+@keyframes panelOpenAnimation {
+  from {
+    width: 0px;
+  }
+
+  to {
+    width: 420px;
+  }
+}
+
+.resources-panel-close {
+  position: absolute;
+  top: 26px;
+  right: 20px;
+  font-size: 18px;
+  background: none;
+  color: var(--vscode-editor-foreground);
+  transition: 200ms all;
+  border: none;
+
+  &:hover {
+    cursor: pointer;
+  }
+}
+
+.resources-panel-header {
+  font-size: 21px;
+  color: #13AA52;
+  text-align: left;
+}
+
+.resources-panel-book-icon {
+  vertical-align: text-top;
+  font-size: 26px;
+  margin-right: 8px;
+}
+
+.resources-panel-links-container {
+  margin-top: 54px;
+}
+
+.resources-panel-link {
+  margin-top: 8px;
+  padding: 10px 20px;
+  background-color: rgba(50, 50, 50, 0.25);
```

Comment:
I'd suggest also using a variable here. With my theme (Material light) it looked pretty strange.
![image](https://user-images.githubusercontent.com/4354632/99506983-1eda2700-2983-11eb-91b5-da3e7e50e2fe.png)


I suggest we might switch to `var(--vscode-sideBar-background)` as background and then use `var(--vscode-sideBar-foreground)` as the new `color`?

With the sidebar variant it looks like this:
![image](https://user-images.githubusercontent.com/4354632/99506918-09fd9380-2983-11eb-8c17-525af67e628c.png)

Reply:
Good call - here's what it looks like now:
<img width="549" alt="Screen Shot 2020-11-18 at 12 34 56 PM" src="https://user-images.githubusercontent.com/1791149/99566691-3ae9c280-299b-11eb-8155-6de859db8d19.png">
<img width="549" alt="Screen Shot 2020-11-18 at 12 34 22 PM" src="https://user-images.githubusercontent.com/1791149/99566694-3b825900-299b-11eb-8d17-8281ce824573.png">
<img width="616" alt="Screen Shot 2020-11-18 at 12 34 07 PM" src="https://user-images.githubusercontent.com/1791149/99566695-3b825900-299b-11eb-8ac5-45b3e4283562.png">